includes/site-health/audit-enqueued-assets/helper.php

```diff
@@ -261,10 +265,12 @@ function perflab_aea_get_total_enqueued_styles() {
 function perflab_aea_get_total_size_bytes_enqueued_styles() {
 	$total_size           = false;
 	$list_enqueued_styles = get_transient( 'aea_enqueued_front_page_styles' );
-	if ( $list_enqueued_styles ) {
+	if ( is_array( $list_enqueued_styles ) ) {
 		$total_size = 0;
 		foreach ( $list_enqueued_styles as $enqueued_style ) {
-			$total_size += $enqueued_style['size'];
+			if ( is_array( $enqueued_style ) && array_key_exists( 'size', $enqueued_style ) && is_int( $enqueued_style['size'] ) ) {
+				$total_size += $enqueued_style['size'];
+			}
```

Comment:
Maybe overly-defensive coding, but in reality a transient could return anything. This ensures we're working with what we expect.